src/classes/AccountDeDuper.cls

```diff
@@ -41,7 +41,7 @@ Public List<Account> SearchAccount (Account pCurrentRecord, String pAccountName,
                     ListOfAdditionalMatches = [select id, Name, BillingStreet, BillingCity, BillingPostalCode, BillingState, Phone, Category__c, Owner.Alias, Record_Type_Name__c from Account where Name LIKE :AccountLikeName and BillingPostalCode=:pPostCode order by Name LIMIT :currentLimit ];
                 }
                 else {
```

Comment:
* This method is empty, I suggest to remove it.
* Methods should use the `lowerCamelCase` naming convention. I suggest to rename accordingly.